modules/dhp_connected_devices/app/services/token_storage_service.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
+class TokenStorageError < StandardError; end
+
+class TokenStorageService
+  include SentryLogging
```

Comment:
This is not needed anymore as far as I can tell. We only log to Sentry in the controller

Reply:
removing